data/processed/hesa/fte_research_students.csv

```diff
@@ -0,0 +1,121 @@
+year,nuts_id,nuts_year_spec,fte_research_students
+2015/16,UKH2,2018,1305.0
```

Comment:
this is still a string, should be number no?

Reply:
Fixed, sorry.